cypress/integration/start.cy.js

```diff
@@ -78,12 +79,12 @@ describe('viewing the start screen', () => {
         const enabledButtons = [
             FILE_MENU_BUTTON_NEW,
             FILE_MENU_BUTTON_OPEN,
-            FILE_MENU_BUTTON_SAVE,
+            FILE_MENU_BUTTON_SAVE_NEW,
         ]
         enabledButtons.forEach(button =>
             expectFileMenuButtonToBeEnabled(button)
         )
-        closeFileMenu()
+        closeFileMenuWithEsc()
```

Comment:
No biggie, but I would consider wrapping this in it's own `it` (example below) or renaming the test to `primary File menu buttons are enabled and close with esc key` to make it obvious for us in the future to figure out why this particular case failed if something was wrong with the esc key press in particular.

> it('File menu is closed with escape', () => {
             closeFileMenuWithEsc()
       })

Same goes for `closeFileMenuWithClick` below.

Reply:
Done